docs/op-stack/src/.vuepress/config.js

```diff
@@ -187,7 +187,8 @@ module.exports = {
         children: [
           '/docs/security/faq.md',
           '/docs/security/policy.md',
-          '/docs/security/pause.md'
+          '/docs/security/pause.md',          
+          '/docs/security/forced-withdrawal.md',
```

Comment:
Not sure, but worth checking: vue might freak out about the trailing comma?

Reply:
No, I verified this works.